packages/lit-starter-js/docs/index.html

```diff
@@ -1,75 +1,96 @@
-
-<!doctype html>
+<!DOCTYPE html>
```

Comment:
Unfortunate that there are so many formatting changes here.

Reply:
I'll revert these